src/routes/blog/[[page]]/+page.svelte

```diff
@@ -246,28 +386,28 @@
                             {#if data.currentPage < data.totalPages}
                                 <a
                                     data-sveltekit-noscroll
-                                    class="flex navigation-button"
+                                    class="navigation-button flex"
                                     href="/blog/{data.currentPage + 1}"
```

Comment:
the pagination will remove `category` or `search` queries 👍🏻 we need to add those here.

feel free to create a helper function that builds a link

Reply:
Done.